File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
       int ind=0;
       for(int j=1;j<nums.length;j++){
            if(nums[ind]!=nums[j]){
                ind++;
                nums[ind]=nums[j];
            }
       }
       return ind+1;
    }
    
}

File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.py
class Solution(object):
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        ind=0;
        for i in range(1,len(nums)):
            if(nums[ind]!=nums[i]):
                ind+=1
                nums[ind]=nums[i]

        return ind+1
            
        